README.md

```diff
@@ -489,51 +489,6 @@ state_map:
 ```
 
 
-## Development
-
-1. Clone this repository into your `config/www` folder using git:
-
-```
-$ git clone https://github.com/kalkih/mini-graph-card.git
-```
-
-2. Add a reference to the card in your `ui-lovelace.yaml`:
-
-```yaml
-resources:
-  - url: /local/mini-graph-card/dist/mini-graph-card-bundle.js
-    type: module
-```
-
-### Instructions
-
-*Requires `nodejs` & `npm`.*
-
-1. Move into the `mini-graph-card` repo, checkout the *dev* branch & install dependencies:
-```console
-$ cd mini-graph-card && git checkout dev && npm install
-```
-
-2. Make changes to the source code.
-
-3. Build the source by running:
-```console
-$ npm run build
-```
-
-4. Refresh the browser to see changes.
-
-    *Make sure cache is cleared or disabled.*
-
-5. *(Optional)* Watch the source and automatically rebuild on save:
-```console
-$ npm run watch
-```
-
-*The new `mini-graph-card-bundle.js` will be build and ready inside `/dist`.*
-
-**If you plan to submit a PR, please base it on the `dev` branch.**
-
```

Comment:
_Getting started_ (as a developer) should remain a part of the general README.md as it's not exclusive to contributing to the project.
Could you revert moving this section?

We should refer to the CONTRIBUTING section here though (as you did before), maybe end with something like this instead?
```
Note that the `dev` branch is the most up-to-date and matches our beta releases.
Please refer to the [Contribution Guidelines](./CONTRIBUTING.md) if you're interested in contributing to the project. (And thanks for considering!)